recipes-devtools/python/python3-jetson-stats_3.1.2.bb

```diff
@@ -0,0 +1,48 @@
+SUMMARY = "Interactive system-monitor process viewer for NVIDIA Jetson Nano, AGX Xavier, TX2, TX1"
+HOMEPAGE = "https://pypi.org/project/jetson-stats/"
+SECTION = "devel/python"
+LICENSE = "AGPL-3.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=8763b57f0092c337eb12c354870a324a"
+
```

Comment:
Could you please add the following line here: 

```
COMPATIBLE_MACHINE = "(tegra)"
```

This is needed to satisfy `yocto_check_layer`

Here is your error:

```
ERROR: Nothing RPROVIDES 'tegra-tools-jetson-clocks' (but /scratch/buildworker/meta-tegra-community-checklayer/build/poky/meta-tegra-community/recipes-devtools/python/python3-jetson-stats_3.1.2.bb RDEPENDS on or otherwise requires it)
tegra-tools RPROVIDES tegra-tools-jetson-clocks but was skipped: incompatible with machine qemux86-64 (not in COMPATIBLE_MACHINE)
NOTE: Runtime target 'tegra-tools-jetson-clocks' is unbuildable, removing...
Missing or unbuildable dependency chain was: ['tegra-tools-jetson-clocks']
```

Reply:
Done. I genuinely thought the whole layer is considered to only be compatible with tegra but now see that most recipes assign `COMPATIBLE_MACHINE`.